en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,68 @@
+``ContentAttribute`` (See `content`_ documentation for more information). The ``ContentAttribute``
+are configured by the `content type`_.
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+
+Sometimes, there is no easy way to render the value to the backoffice user.
+A custom ``ContentAttribute`` could be an array, which cannot be displayed directly in javascript.
+In that way the ``ContentAttribute`` class provide a specific attribute ``stringValue`` which is the
+representation of the ``value`` in an HTML string.
+
+Transformers
+------------
+
+Open Orchestra uses a ``ValueTransformer`` strategy manager to generate the HTML strings for
+multiple ``value`` type. Open Orchestra already provides some strategies located in
+``OpenOrchestra\Backoffice\ValueTransformer\Strategies``.
+
+Strategy creation
+~~~~~~~~~~~~~~~~~
+
+Service used as ``ValueTransformer`` must be tagged as ``open_orchestra_backoffice.value_transformer.strategy``.
+
+.. code-block:: yaml
+
+    tags:
+        - { name: open_orchestra_backoffice.value_transformer.strategy }
+
+Such services also need to implement ``OpenOrchestra\Backoffice\ValueTransformer\ValueTransformerInterface``.
+
+Strategy access
+~~~~~~~~~~~~~~~
+
+Open Orchestra uses the ``OpenOrchestra\Backoffice\ValueTransformer\ValueTransformerManager``
+as an easy way to access every transformation strategies.
+All ``ValueTransformer`` strategies are registered in the ``ValueTransformerManager``.
+The ``transform`` method of the manager:
+ - searches the first transformation strategy supporting the current ``fieldType`` and ``value`` of the ``ContentAttribute``
+ - uses this strategy's ``transform`` method to return the ``stringValue``
+
+If no transformer supports the actual parameters, the method returns the original ``value``.
+
+Examples
+--------
+
++------------------------------+-------------------------------------------+
+| Value                        | StringValue                               |
++==============================+===========================================+
+| "string"                     | "string"                                  |
++------------------------------+-------------------------------------------+
+| 1                            | "1"                                       |
++------------------------------+-------------------------------------------+
+| null                         | "none"                                    |
++------------------------------+-------------------------------------------+
+| Object()                     | Object::__toString()                      |
++------------------------------+-------------------------------------------+
+| array('foo' => array('bar')) | '<ul><li><ul><li>bar</li></ul></li></ul>' |
++------------------------------+-------------------------------------------+
```

Comment:
Very nice: can you add a column indicating which transformer is used for the generation ?